src/tyr/CMakeLists.txt

```diff
@@ -4,16 +4,16 @@ set(sources
   actor.cc
   height_serializer.cc
   isochrone_serializer.cc
+  locate_serializer.cc
+  matrix_serializer.cc
+  route_serializer.cc
   serializers.cc
+  trace_serializer.cc
   transit_available_serializer.cc)
 
 set(sources_with_warnings
-  locate_serializer.cc
-  matrix_serializer.cc
-  route_serializer.cc
   route_serializer_osrm.cc
```

Comment:
I could use feedback on these last two files.

The error is an unused function in each file. I don't want to delete them unless that's desired by the project.

Reply:
Which function is it @dillona ? Can you quickly point us there so no one has to run it? Thanks:)

Reply:
Yes of course.

It is the serialize function. See https://github.com/valhalla/valhalla/blob/d44cd26f756e3861f0fc01b86e2a3b8bd2435f6c/src/tyr/route_serializer_osrm.cc#L1662